vpr/src/place/place_util.cpp

```diff
@@ -65,6 +65,7 @@ static GridBlock init_grid_blocks() {
 void t_placer_costs::update_norm_factors() {
     if (place_algorithm.is_timing_driven()) {
         bb_cost_norm = 1 / bb_cost;
+        cong_cost_norm = 1/ cong_cost;
```

Comment:
What if place_algorithm is not congestion aware?
Is it possible to divide by zero?